packages/library/src/components/bal-radio/bal-radio.tsx

```diff
@@ -22,6 +22,11 @@ export class Radio {
    */
   @Prop() interface: 'radio' | 'select-button' = 'radio'
 
+    /**
+   * The tabindex of the control.
+   */
+  @Prop() balTabindex: number = 0
```

Comment:
maybe to change balTabindex -> balTabIndex

Reply:
tabindex is one word. I thought the same.

Reply:
ohh, ok than